snapshot/localmounter_windows.go

```diff
@@ -46,18 +48,51 @@ func (lm *localMounter) Mount() (string, error) {
 		// The Windows snapshotter does not have any notion of bind mounts. We emulate
 		// bind mounts here using the bind filter.
 		if err := bindfilter.ApplyFileBinding(dir, m.Source, m.ReadOnly()); err != nil {
-			return "", errors.Wrapf(err, "failed to mount %v: %+v", m, err)
+			return "", errors.Wrapf(err, "failed to mount %v", m)
 		}
 	} else {
 		if err := m.Mount(dir); err != nil {
-			return "", errors.Wrapf(err, "failed to mount %v: %+v", m, err)
+			if strings.Contains(
+				err.Error(),
+				"cannot access the file because it is being used by another process",
+			) {
+				// see https://github.com/moby/buildkit/issues/5807
+				// if it's a race condition issue, do max 2 retries with some small backoff
+				// should adjust the retries if this persists but 1 retry
+				// seems to be enough.
+				if err := mountWithRetries(m, dir, 2); err != nil {
```

Comment:
why not just call `mountWithRetries` directly to avoid duplicate `m.Mount` calls

Reply:
I'd considered that but thought it but wanted to make the call to `mountWithRetries` only explicit on cases where we've seen this race failure. And also once we find the better fix, we can just clean up by removing this if block.
However, looking at it again, I think the duplication screams, let me fix that.

Reply:
Fixed.